contracts/util/FleekStrings.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.7;
+
+import "../FleekERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/utils/Base64.sol";
+import "./FleekSVG.sol";
+
+library FleekStrings {
+    using Strings for uint256;
+    using Strings for uint160;
+    using FleekStrings for bool;
+
+    /**
+     * @dev Converts a boolean value to a string.
+     */
+    function toString(bool _bool) internal pure returns (string memory) {
+        return _bool ? "true" : "false";
+    }
+
+    /**
+     * @dev Converts a string to a base64 string.
+     */
+    function toBase64(string memory str) internal pure returns (string memory) {
+        return Base64.encode(bytes(str));
+    }
+
+    /**
+     * @dev Converts FleekERC721.App to a JSON string.
+     * It requires to receive owner address as a parameter.
+     */
+    function toString(FleekERC721.App storage app, address owner) internal view returns (string memory) {
```

Comment:
Should we copy the tokenId owner to the app struct during minting/transfer? This way this library can just pull from the App input.  Otherwise someone can call this with a random owner and get a URI that's not right.  It's an edge case so your decision in the end @zoruka.  Cuz yes, why would someone do that? So more a question of fool proofing it.

Reply:
Hmmm I'm thinking about it, since the owner is stored in a struct aside of the app metadata, we would need to also update the metadata every time that mint or transfer is run and enforce that this field is updated always. It seems a good idea for this scenario, but still have the tradeoff of make sure that it is always synced. Maybe I could try build this function in another way. But anyways, since this is a internal function it would be wrong just in case of bad development implementation and seems like users could not trick it.

Reply:
Got it, will create a future card for it.